### DISCUSSION TITLE
I can't make GHelper to stop putting a limit in the charge battery.

### DISCUSSION THREAD

**Discussion by MemoSainz** [seerge/g-helper#2890] [General]
Hello. I was desiring another app (MyAsus) to set the limit. But btw, every time I want to delete from the Json "charge limit" it appears again when I open GHelper.

I admire the work behind GHelper and it's helping me a lot, I just desire a way or a button or something that GHelper let another app to put the limit in my batter limit. As I care about the battery life.

Thank you guys for this great app!

**Comment by seerge** [comment]
@MemoSainz hello, 

You need to quit app first, then delete ``"charge limit" : 80,`` line completely from config. And start app again. Then it will ignore limit unless you drag / interact with charge slider again.

Why don't you just set same charge limit ? Both apps set charge limit in exactly same way, via asus endpoint.
